Detourist/Detourist/SearchViewController.swift

```diff
@@ -13,7 +13,7 @@ import GooglePlaces
 
 class SearchViewController: UIViewController, CLLocationManagerDelegate, GMSAutocompleteResultsViewControllerDelegate {
     
-    @IBOutlet weak var mapView: GMSMapView!
+    var mapView: GMSMapView?
```

Comment:
This was the biggest problem - when `mapView` was declared as `GMSMapView!` with the exclamation point, it meant that `mapView` had to always be non-nil. Then you'd get a crash in `viewDidLoad` even if you just tried to assign to `self.mapView`, because first there'd be a check to see if `self.mapView` was nil.